### PR TITLE
Add more details to the readme and fix language

### PR DESCRIPTION
**Pull request by jukoson**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,12 +1,11 @@
...
 
-The configuration file and python file needed to read data from a local
-FPGA
+The configuration file and python file needed to read out and display statistics for FPGAs in the system
 
 
 # Prerequisites
 
-- psycopg2
+- Python2 package psycopg2. Install eg. with `python -m pip install psycopg2`
```
Comment (afswarm) [inline README.md]: Put "psycopg2" in quotes or code quotes to make it clear?

Comment (jukoson) [reply README.md]: I made it italic, that good ?

Comment (afswarm) [reply README.md]: ok

Comment (sdressler) [reply README.md]: @jukoson @afswarm  Actually, sorry for being late to the party, but AFAIK it is now `pip install psycopg2-binary`. Or even preferred use `yum install python-psycopg2`.

Comment (jukoson) [reply README.md]: All of the commands work for me, including python -m pip... 
I am fine with yum too. Both of you ok to switch it to yum in this case ?

**Comment by sdressler** [comment]
It’s more, that a `pip install psycopg2` might fail because it requires dev packages for postgres. I just stumbled over this. Yum is the best choice IMO.

--

Sebastian Dressler
Solution Architect
+49 30 994 0496 72 | sebastian@swarm64.com
Zoom in at 290 044 7969

Swarm64 AS
Parkveien 41 B | 0258 Oslo | Norway

Registered at Brønnøysundregistrene in Norway under Org.-Number 911 662 787 CEO/Geschäftsführer (Daglig Leder): Thomas Richter; Chairman/Vorsitzender (Styrets Leder): Dr. Sverre Munck


Swarm64 AS Zweigstelle Hive

Ullsteinstr. 120 | 12109 Berlin | Germany Registered at Amtsgericht Charlottenburg - HRB 154382 B

________________________________

**Pull request merged**